templates/plugin-info/description.php

```diff
@@ -32,7 +33,11 @@
 				<?php endfor ?>
 			</ul>
 		</div>
-	<?php endif ?>
+	<?php
+	endif;
+	
+	if ( ! empty( $plugin->info->description ) ) :
```

Comment:
This new logic prevents an empty `<div>      </div>` from appearing as the description when there's no description to show.